docs/ref/django-admin.txt

```diff
@@ -776,6 +776,11 @@ Enables fixing of migration conflicts.
 Allows naming the generated migration(s) instead of using a generated name. The
 name must be a valid Python :ref:`identifier <python:identifiers>`.
 
+.. django-admin-option:: --no-header
```

Comment:
These options could have a `.. versionadded:: 2.2` directive. If you search this file, you see some similar example uses.

Reply:
I can add that, I wasn't aware of it -- thanks!